practice/practice-02/src/sass/layout/_navbar.scss

```diff
@@ -0,0 +1,158 @@
+.icon-menu {
```

Comment:
For navbar but I saw you don't style for only navbar. This style can be apply for all icon-menu

Reply:
I don't know what you mean, can you explain more about it?

Reply:
For navbar style, that mean all style off this file should only effect to navbar only. But in your component, it also style for general style, not only for navbar